conftest.py

```diff
@@ -1,6 +0,0 @@
-pytest_plugins = [
-    "tests.fixtures.appclient",
-    "tests.fixtures.entities",
-    "tests.fixtures.simulations",
-    "tests.fixtures.taxbenefitsystems",
```

Comment:
I am not sure I understand why this was deleted.

Reply:
Because otherwise we can't predictably test the built packages in an isolated way.

PyTest decided to forbid users to have more than one conftest per project. I don't know why they did that, but it forces us to remove it if we want isolation as we actually ship two packages independently, core and web-api.

One alternative is to ship all together, which makes a bit more sense.

Today we have:

```
openfisca_core
openfisca_web_api
tests (not shipped)
```

With this PR we will have:

```
openfisca_core
openfisca_core.tests
etc.
```

In order for this file to be kept, we would have to do something like

```
src/openfisca_core
str/openfisca_web_api
```

It is my preferred approach, but it seemed too much of a change.